### ISSUE TITLE
[Bug]: Possible Menubar icon disappears when NORD VPN started?

### ISSUE THREAD

**Issue by scschulte** [jordanbaird/Ice#473]
### Check Existing Issues

- [X] I have checked existing issues, and this issue is not a duplicate

### Description

When I activated NORD VPN I noticed that SugarSync was somehow no longer appearing in the menubar.  When I then turned-off my VPN I had to sign-in again to SugarSync.  Is this possibly a bug with ICE?

### Steps to Reproduce

1. Start NORD VPN
2. Notice SugarSync closes and is no longer in the menubar
3. Try to move from menubar setting VISIBLE to HIDDEN and message says it is not valid
4. Close NORD VPN and
5. re-install SugarSync
6. Icon is now in menubar -- but afraid it will disappear if I turn-on NORD VPN again!

### Ice Version

0.11.12

### macOS Version

14.7.1

**Comment by jordanbaird** [comment]
I don't think this is related to Ice. Sounds like SugarSync might be crashing when the VPN is enabled.
